cdap-standalone/src/main/java/co/cask/cdap/StandaloneMain.java

```diff
@@ -69,6 +69,8 @@
 import co.cask.cdap.metrics.query.MetricsQueryService;
 import co.cask.cdap.notifications.feeds.guice.NotificationFeedServiceRuntimeModule;
 import co.cask.cdap.notifications.guice.NotificationServiceRuntimeModule;
+import co.cask.cdap.operations.OperationalStatsLoader;
```

Comment:
These two classes are in ``cdap-app-fabric``, not in ``cdap-operational-stats-core``. We shouldn't need the dependency I added here - https://github.com/caskdata/cdap/pull/7479#discussion_r93088047. Testing it.